### PR TITLE
rope climbing robot 

### PR DESCRIPTION
**Pull request by adityatidke**

### CODE REVIEW

**Comment by adityatidke** [comment]
Please merge this .

**Review by SanjeevKrishnan — CHANGES_REQUESTED** [review]
@adityatidke Changes are required!
1. Folder names given in the documentation guideline has to be followed properly(with Case Sensitivity). For ex., it is "Images and Videos", not "images and videos".
2. flowchart.jpg has to be present inside Images and Videos/Images.
3. Snapshots of all CAD files have to be present inside the Mechanical Design folder.
4. Names of files have to be self-explanatory. Few files inside the Mechanical Design folder have names as "WhatsApp Images" which is not allowed.
5. CAD images have to be present inside the Mechanical Design folder, not Images and Videos/Images. All other project images related images like prototype images, flowcharts, circuit diagrams, team picture has to be present inside Images and Videos/Images.
6. Text in poster, report, and README.md has to be justified. You can justify text in README.md using HTML tags.

**Comment by agshubhs** [comment]
@adityatidke Use Markdown Syntax it would be easier to add the links, images, list in the Readme file.
https://commonmark.org/help/
And also add real photos in readme

**Comment by adityatidke** [comment]
Real image of the robot is added      
 @Shubhanshu07 sir, how can we align the text in markdown ?

**Comment by agshubhs** [comment]
@adityatidke you can use p tag to align justify for each paragraph.
For list you can use markdown syntax

**Comment by adityatidke** [comment]
Thank you sir . I have updated the readme according

File: readme.md [hunk 1]
```diff
@@ -0,0 +1,148 @@
+ <p align="justify">
+ <h1>Rope-Climbing-Robot</h1>
+Srishti 2020
+</p>
+
+
+<p align="justify">
+<h2>Abstract</h2>
```
Comment (SanjeevKrishnan) [inline readme.md]: @adityatidke Leave Heading while using tag. For example,
```
## Abstract
<p align="justify">This Rope climbing bot is used to climb ropes in an easy and efficient manner to reduce manual labour.Motorised rollers are used for climbing and a leadscrew is used for the gripping.</p>
```

File: readme.md [hunk 2]
```diff
@@ -0,0 +1,148 @@
...
+- The load carrying capacity can be improved by using stronger motors and parts.
+- Image detection software can be incorporated with the help of a camera , so that it can detect a rope on its own and climb it.
+- Modification in the design can be made so that it can measure its diameter of the rope without prefeding the data into the encoder.
+- Pneaumatics can be introduced in the the model to improve the gripping and climbing speed.
+
+<p align="justify">
+<h2>Team Member</h2>
+</P>
+
+- <a href="https://github.com/adityatidke">Aditya Tidke</a>
```
Comment (SanjeevKrishnan) [inline readme.md]: Don't use HTML tags for Team Members, Mentors list, Reference. And make it ordered list

**Review by SanjeevKrishnan — CHANGES_REQUESTED** [review]

**Comment by adityatidke** [comment]
Suggestions have been implemented .

**Comment by agshubhs** [comment]
@adityatidke Great work PR looks good to me 👍

**Pull request merged**